### DISCUSSION TITLE
Turn left and right

### DISCUSSION THREAD

**Discussion by hazey-orion** [mrbungle64/ecovacs-deebot.js#465] [Q&A]
Hi,
I've seen from the documentation that MoveLeft and MoveRight are deprecated.
Is there any command that replaces these or any alternative?
Thanks!

**Comment by hazey-orion** [comment]
Nvm, I see there is a current discussion for this :D

**Reply by mrbungle64** [reply]
@emma-sudds 

Can you please tell me for what use case you need the turn left and right commands?
And also what model you own.

> I see there is a current discussion for this

Do you mean [this issue](270)?

**Reply by hazey-orion** [reply]
I want to be able to manually turn the deebot with a ps4 controller (which I have successfully done with forward and back) but not turning/pivoting. 
Yes, that's the issue.

**Reply by mrbungle64** [reply]
> I want to be able to manually turn the deebot with a ps4 controller (which I have successfully done with forward and back) but not turning/pivoting.

I like your idea 👍🏻 

What model do you own?

**Reply by hazey-orion** [reply]
Thanks! Born from my frustration with the bot. The T9 I believe!

**Reply by hazey-orion** [reply]
I think I saw in the other thread that you captured the traffic of the move commands.
I've been looking into trying to do this myself but couldn't see the traffic on wireshark.
Do you know if I need monitor mode on my network adpater? I've been getting mixed answers online.

**Reply by mrbungle64** [reply]
@emma-sudds 
I used [mitmproxy](https://mitmproxy.org/) to capture the traffic.
I'm not sure if it also works with wireshark.

**Reply by hazey-orion** [reply]
Sorry for the late response. 
Nice! Thanks for the info. I'll give it a try.
I'm learning a lot :D

**Reply by hazey-orion** [reply]
Maybe you can only see commands if you can control the robot manually via the app but I can't see anything relating to bot movement in the traffic :(

**Reply by mrbungle64** [reply]
> Maybe you can only see commands if you can control the robot manually via the app

Yes, that's the problem 😉
